about.html

```diff
@@ -0,0 +1,182 @@
+    />
+    <link rel="stylesheet" href="./assets/about.css" />
+    <title>Erriecha Annual | Festival 2023</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar-top bg-second">
+        <ul class="navtop-container start-align end-align">
+          <li class="navtop-list">
+            <a class="navtop-link txt-main">
+              <img src="./assets/images/facebook.png" class="navtop-image" />
+            </a>
+          </li>
+          <li class="navtop-list txt-main">
+            <a class="navtop-link">
+              <img src="./assets/images/twitter.png" class="navtop-image" />
+            </a>
+          </li>
+          <li class="navtop-list txt-main">
+            <a class="navtop-link txt-main"> English </a>
+          </li>
+          <li class="navtop-list txt-main">
+            <a class="navtop-link txt-main"> Mypage </a>
+          </li>
+          <li class="navtop-list">
+            <a class="navtop-link txt-main"> Logout </a>
+          </li>
+        </ul>
+      </nav>
+      <nav class="navbar-main bg-white">
+        <ul class="navmain-icon start-align">
+          <li class="navlist-icon listevent-logo">
+            <a href="./index.html" class="listicon-link listicon-image">
+              <img class="linklogo-image" src="./assets/images/titleicon.png" />
+            </a>
+          </li>
+          <li class="navlist-icon listicon-close">
+            <a href="" class="listicon-link txt-second linkicon-close"
+              >&#215;</a
+            >
+          </li>
+         
+        </ul>
+        <ul class="navmain-container end-align">
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-third">About</a>
+          </li>
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-second">Program</a>
+          </li>
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-second">Join</a>
+          </li>
+          <li class="navmain-list">
+            <a href="" class="navmain-link txt-second">Sponsor</a>
+          </li>
+          <li class="navmain-list list-news">
+            <a href="" class="navmain-link txt-second">News</a>
+          </li>
+          <li class="navmain-list list-campaign">
+            <a href="" class="navmain-link navmain-name txt-third">
+              Erriecha
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="head-section">
+        <div class="breadcrumb-container start-align end-align">
+          <ul class="crumb-orderlist">
+            <li class="crumb-list">Home &#62;</li>
+            <li class="crumb-list">About &#62;</li>
+            <li class="crumb-list">Erriecha</li>
+          </ul>
+          </div>
+        <div class="headsection-container start-align end-align">
+          <div class="head-humberger mobile-only">
+            <a href="" class="linkicon-humbergur"> &#9776;</a>
+          </div>
```

Comment:
- [ ] Your hamburger is not working at this moment. Kindly make it to work.

[your-hamburger.webm](https://user-images.githubusercontent.com/26597432/226097414-79a55f38-510a-4879-8e4c-9e4f77d9841c.webm)